libs/membership/api/src/lib/subscription-payments/subscription-payments.service.ts

```diff
@@ -103,6 +103,9 @@ export class SubscriptionPaymentsService {
         // skip invoices where the subscription has been deleted
         subscriptionID: {
           not: null
+        },
+        subscription: {
+          confirmed: true
```

Comment:
You could maybe write a testcase here to test that if confirmed =  false its not returned